src/middlewares/verifyToken.js

```diff
@@ -0,0 +1,21 @@
+import JWT from 'jsonwebtoken';
+import models from '../models';
+
+export default async (req, res, next) => {
+  const { headers: { token } } = req;
+  if (!token || token === '') {
+    return res.status(400).json({ status: 400, error: 'Please provide a token.' });
+  }
+  try {
+    const { sub } = await JWT.verify(token, process.env.SECRET);
+    const { id } = sub;
+    const { dataValues: user } = await models.Users.findOne({ where: { id } });
```

Comment:
@eokwukwe The reason for this query here is a security measure. To make sure that a user with this id exists in our database and to detect it early here. This prevents a user that is not in our database, or maybe that used to be but has been deleted, from accessing the protected routes. Do I still remove it?

Reply:
> @eokwukwe The reason for this query here is a security measure. To make sure that a user with this id exists in our database and to detect it early here. This prevents a user that is not in our database, or maybe that used to be but has been deleted, from accessing the protected routes. Do I still remove it?

You need to implement it in a way that it can be used by others.